src/database/models/flextime.ts

```diff
@@ -0,0 +1,9 @@
+/**
+ * What Flextime returns from Severa.
+ */
+interface Flextime {
+  totalFlextimeBalance: number | null;
```

Comment:
Why is this added here ? It should already be in develop ?

Reply:
hmmm... i actually have no idea